css.md

```diff
@@ -52,6 +53,18 @@ This style guide focusses primarily on the usage of SASS, which is the standard
     :font-size 0.3em
   ```
 
+* Top-level numeric calculations should always be wrapped in parentheses
+
+  ```sass
+  // good
+  .foo
+    width: (100% / 3)
+
+  // bad
+  .foo
+    width: 100% / 3
+  ```
```

Comment:
This is like saying you should "always use parenthesis in ruby"  there's a reason this shit is optional in the language.